index.html

```diff
@@ -3,76 +3,96 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sapota UI | Typography</title>
+    <link rel="icon" href="./assets/sapota-icon.png" type="image/x-icon" />
+    <link rel="stylesheet" href="./style.css" />
     <link rel="stylesheet" href="./doc.css" />
+    <link rel="stylesheet" href="./docs/css/doc.css" />
     <link rel="stylesheet" href="./docs/css/index.css" />
-    <title>Sapota UI| Home</title>
   </head>
   <body>
-    <header class="top-nav">
+    <header class="top-nav box-shadow main-header">
       <nav class="navigation">
-        <div class="nav-brand">SAPOTA UI</div>
-        <ul class="list-non-bullets right">
+        <div class="logo-btn-header">
+          <div class="nav-brand">SAPOTA UI</div>
+          <button class="menu-icon">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+          </button>
+        </div>
+        <ul id="main-menu" class="list-non-bullets nav-right">
           <li class="list-item-inline">
             <a class="link link-active" href="/">Home</a>
           </li>
           <li class="list-item-inline">
-            <a class="link" href="projects.html">Docs</a>
+            <a class="link" href="./docs/avatar.html">Docs</a>
           </li>
           <li class="list-item-inline">
-            <a class="link" href="blog.html">Github</a>
+            <a class="link" href="https://github.com/abhij1607">Github</a>
           </li>
         </ul>
       </nav>
-      <button class="nav-right">Theme</button>
```

Comment:
Maybe your forgot to add this button.

Reply:
No, I removed the theme option for now due to time constraints but thank you for pointing this...it will remind me to add it later on